spaceship/lib/spaceship/two_step_client.rb

```diff
@@ -35,7 +35,7 @@ def handle_two_step(response)
         result = choose(*available)
         device_id = result.match(/.*\t.*\t\((.*)\)/)[1]
         select_device(r, device_id)
-      elsif r.body.kind_of?(Hash) && r.body["phoneNumberVerification"].kind_of?(Hash)
+      elsif r.body.kind_of?(Hash) && r.body["trustedPhoneNumbers"].kind_of?(Array) && r.body["trustedPhoneNumbers"].first.kind_of?(Hash)
```

Comment:
But these values in the body are never actually used, right?  So couldn't we just delete these tests here, or else update the test to check `r.body["securityCode"]` in order to support line 51 below?  (previously this line was testing the response to see if line 51 could execute properly)